app/src/main/java/ch/sdp/vibester/database/UsersRepo.kt

```diff
@@ -50,17 +64,48 @@ class UsersRepo @Inject constructor() {
      * @param callback function to be called when the the user has been created
      */
     fun createUser(email: String, username: String, handle: String, callback: (String) -> Unit) {
-        var newUser = User(handle, username, "", email, 0, 0, 0, 0
-        )
-
+        var newUser = User(handle, username, "", email, 0, 0, 0, 0)
         val newId = Util.createNewId()
-
         dbRef.child(newId).setValue(newUser)
             .addOnSuccessListener {
                 callback(email)
             }
     }
 
+    /**
+     * Search for users by its any field in Firebase Realtime Database
+     * @param field user fields used for a search
+     * @param searchInput search text inputed by user
+     * @param callback function to call with found users by username
+     *
+     * Comment about \uf8ff:
```

Comment:
I suggest putting this comment right below line 89 so that when people read the code, it's easier for them to find the explanation.

Reply:
done!